site/apache.md

```diff
@@ -0,0 +1,460 @@
+
+# Load Balancing with Apache Web Server
+
+This page describes how to setup and start a Apache Web Server for load balancing inside a Kubernets cluster. The configuration and startup can either be automatic when you create a domain using the WebLogic Operator's `create-weblogic-domain.sh` script, or manually if you have an existing WebLogic domain configuration.
+
+## Build Docker Image for Apache Web Server
+
+You need to prepare the Docker image for Apache Web Server that enbeds Oracle WebLogic Server Proxy Plugin.
+
+  1. Download and build the Docker image for the Apache Web Server with 12.2.1.3.0 Oracle WebLogic Server Proxy Plugin.  See the instructions in [Apache Web Server with Oracle WebLogic Server Proxy Plugin on Docker](https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-webtier-apache).
+
+  2. tag your Docker image to `store/oracle/apache:12.2.1.3` using `docker tag` command.
+
+```
+
+     $ docker tag 12213-apache:latest store/oracle/apache:12.2.1.3
+
+```
+
+More information about the Apache plugin can be found at: [Apache Web Server with Oracle WebLogic Server Proxy Plugin on Docker](https://docs.oracle.com/middleware/1213/webtier/develop-plugin/apache.htm#PLGWL395).
+
+Once you have access to the Docker image of the Apache Web Server, you can go ahead follow the instructions below to setup and start Kubernetes artifacts for Apache Web Server.
+
+
+## When use Apache load balancer with a WebLogic domain created with the WebLogic Operator
+
+Please refer to [Creating a domain using the WebLogic Operator](creating-domain.md) for how to create a domain with the WebLogic Operator.
+
+You need to configure Apache Web Server as your load balancer for a WebLogic domain by setting the `loadBalancer` option to `APACHE` in `create-weblogic-domain-inputs.yaml` (as shown below) when running the `create-weblogic-domain.sh` script to create a domain. 
+
+```
+
+# Load balancer to deploy.  Supported values are: VOYAGER, TRAEFIK, APACHE, NONE
+
+loadBalancer: APACHE
+
+```
+
+The `create-weblogic-domain.sh` script installs the Apache Web Server with Oracle WebLogic Server Proxy Plugin into the Kubernetes *cluster*  in the same namespace of the *domain*.
+
+The Apache Web Server will expose a `NodePort` that allows access to the load balancer from the outside of the Kubernetes cluster.  The port is controlled by the following setting in `create-weblogic-domain-inputs.yaml` file:
```

Comment:
allows access to the load balancer from outside of the Kubernetes cluster.
The port is configured by setting 'loadBalancerWebPort'  in `create-weblogic-domain-inputs.yaml` file.